src/components/molecules/PollBox/PollBox.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { useCallback, useMemo } from "react";
+import { useForm, useFieldArray } from "react-hook-form";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faPaperPlane, faPlus } from "@fortawesome/free-solid-svg-icons";
+
+import { MAX_POLL_CHOICES } from "settings";
+
+import { getVenueOwners } from "api/admin";
+
+import { PollValues, PollQuestion } from "types/chat";
+
+import { InputField } from "components/atoms/InputField";
+
+import { useVenueId } from "hooks/useVenueId";
+import { useRoles } from "hooks/useRoles";
+import { useUser } from "hooks/useUser";
+
+import "./PollBox.scss";
+
+export interface PollBoxProps {
+  createPoll: (props: PollValues) => void;
+}
+
+const DEFAULT_QUESTION: Partial<PollQuestion> = { name: "" };
+const DEFAULT_VALUES = {
+  topic: "",
+  questions: [DEFAULT_QUESTION, DEFAULT_QUESTION],
+};
+
+export const PollBox: React.FC<PollBoxProps> = ({ createPoll }) => {
+  const venueId = useVenueId();
+  const { userRoles } = useRoles();
+  const { user } = useUser();
+
+  const userId = user?.uid;
+  const isAdmin = Boolean(userRoles?.includes("admin"));
+
+  const { register, control, handleSubmit, reset, watch } = useForm<PollValues>(
+    {
+      defaultValues: DEFAULT_VALUES,
+    }
+  );
+  const { fields, append } = useFieldArray({ name: "questions", control });
+  const [question1, question2] = watch("questions");
+  const topic = watch("topic");
+
+  const onPollSubmit = handleSubmit(async ({ topic, questions }) => {
+    if (!venueId || !userId) return;
+
+    const owners = await getVenueOwners(venueId);
+
+    const poll = {
+      canBeDeleted: isAdmin && owners.includes(userId),
```

Comment:
This information should not be here. remove this line. It doesn't make sense. What you do is you allow people to delete the Polls of admins. What we want to have is to allow admins delete every poll

Reply:
> What you do is you allow people to delete the Polls of admins.

@mike-lvov I don't know that that is true?

`const { userRoles } = useRoles();`
`const isAdmin = Boolean(userRoles?.includes("admin"));`
`const owners = await getVenueOwners(venueId);`

To me that reads as the poll `canBeDeleted` if the current user is an admin and an owner.

In an earlier comment I believe you said that to be able to delete you need to be both an admin and an owner.

So I think the logic here is correct?

Reply:
@0xdevalias 
This `poll` with `canBeDeleted` is passed to `createPoll` a few lines below

Reply:
Fair enough. Missed that aspect.

Reply:
fixed in 55c674d712fa3c88df459d4ed1c2a6b24482e632